data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('1', 'Agumon', '2020-2-3', '0', '1', '10.23');
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('2', 'Gabumon', '2018-11-15', '2', '1', '8');
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('3', 'Pikachu', '2021-1-7', '1', '0', '15.04');
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('4', 'Devimon', '2017-5-12', '5', '1', '11');
```

Comment:
- [ ] I think it would be better if you use ``true`` or ``false`` values(instead of `0` and `1` values) on the `neutered` column as it is of type `boolean` to make your database more readable. I know all of them will give the same result, but I think this approach will be more readable